phoenix-core/src/main/java/org/apache/phoenix/query/ConnectionQueryServicesImpl.java

```diff
@@ -1336,6 +1337,8 @@ private TableDescriptor ensureTableCreated(byte[] physicalTableName, PTableType
                         PBoolean.INSTANCE.toObject(newDesc.build().getValue(MetaDataUtil.IS_LOCAL_INDEX_TABLE_PROP_BYTES)))) {
                     newDesc.setRegionSplitPolicyClassName(IndexRegionSplitPolicy.class.getName());
                 }
+                // disable split policy for SYSTEM.TASK
+                isSplitPolicyUpdatedForTaskTable(newDesc);
```

Comment:
Since we have the namespace mapping enabled property available at this point, can't we get the exact expected name i.e. SYSTEM.TASK vs SYSTEM:TASK instead of trying both variations inside this method?